reframe/frontend/executors/policies.py

```diff
@@ -349,12 +349,13 @@ def exit(self):
 
                 _cleanup_all(self._retired_tasks, not self.keep_stage_files)
                 if self._pipeline_statistics:
-                    new_num_retired = len(self._retired_tasks)
+                    num_retired_actual = len(self._retired_tasks)
```

Comment:
This is not the `num_retired_actual`. You should do the subtraction here instead.

Reply:
I fixed that.